src/purposeSubCategories/other.ts

```diff
@@ -0,0 +1,12 @@
+import { makeEnum } from '@transcend-io/type-utils';
+
+export const OtherPurposeSubCategory = makeEnum({
+  /** Fallback sub purpose */
+  Other: 'Other',
+});
```

Comment:
I don't think we need a fallback subcategory for the `Other` subpurpose. Fallback subcategories are for when we want to assign a subdatapoint to an explicit parent category (i.e. not `Other`) but no subcategory in particular.

Reply:
yeah @LuGary i noticed after the fact that you didn't add one for it for the data sub categories. Can remove, but it is currently a tag/label here https://github.com/transcend-io/privacy-types/blob/main/src/objects.ts#L28

so should we remove that as well?

Reply:
No I think having a top-level `OTHER` parent category is fine, it's just that the `OTHER` parent category won't have a fallback subcategory like for the other parent categories, because we will never assign a subdatapoint to the `OTHER` parent category without a subcategory in mind. For example, we might want to assign a subdatapoint to `FINANCIAL`, but not know exactly what type of financial data it is, so it will be set to the fallback `FINANCIAL` subcategory. This won't happen for the `OTHER` parent category

Reply:
@LuGary sure, but I thought we still need a row in the subcategory table bc all the assigned categories/purposes should be stored in the `SubDataPointDataSubCategory` or `SubDataPointProcesingPurposeSubCategory` table?

Reply:
Yes, but those rows will be user-generated, there won't be a default one we create for the `OTHER` parent category. Basically every category except for `OTHER` will have at least one Transcend-created subcategory, which is what we'll assign subdatapoints to.